### DISCUSSION TITLE
Remove the Settings Box at the Top Right of the Screen when in Tribe Mode

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#2163] [Ideas] [answered]
When in tribe mode It doesn't make sense to have the settings box at the top when the main settings are in the center of the screen.

<img width="1552" alt="Screen Shot 2021-12-09 at 11 39 11 AM" src="https://user-images.githubusercontent.com/83455454/145447890-e7566139-2987-4701-96a1-e594b30e773d.png">

**Comment by Miodec** [comment, marked as answer]
I think the new version hides that menu already.

**Reply by tajhans** [reply]
Are you sure? It's not in the current dev site.

**Reply by Miodec** [reply]
I am. Current dev has old code.

**Reply by tajhans** [reply]
@Miodec Ah ok.
